public/app/plugins/datasource/graphite/components/helpers.tsx

```diff
@@ -20,3 +21,42 @@ export function mapFuncDefsToSelectables(funcDefs: FuncDefs): Array<SelectableVa
 
   return sortBy(categories, 'label');
 }
+
+function createEditableParam(paramDef: ParamDef, value: string | number): EditableParam {
+  return {
+    name: paramDef.name,
+    value: value?.toString() || '',
```

Comment:
Oh good point. The value actually can be `undefined` (it's picked from an array and in some cases provided array's index may be out of array's length). TS doesn't complain about it because we would need `noUncheckedIndexedAccess` enabled I guess. I'll update the type to reflect it. It's already covered by tests (most of them would fail if it was `value.toString()`)

- [ ] update type definitions

Reply:
19d7464442e909257dfac2f0e21b2fb5a58aecb2